docs/what-are-product-design-sprints.md

```diff
@@ -0,0 +1,34 @@
+---
+id: product-design-sprint
+title: What are Product Design Sprints?
+sidebar_label: What are Product Design Sprints?
+---
+
+All our projects are design-led, starting with post-its and sketching, ending
```

Comment:
'design-led' is not a word, recommended 'design-centric'

Reply:
Fixed in bb2c077